src/lt/util/js.cljs

```diff
@@ -1,37 +1,70 @@
 (ns lt.util.js
-  "Provide misc js related fns")
+  "Provide misc Javascript related functions.")
 
-(defn every [ms func]
+(defn every
+  "Execute `func` every `ms` milliseconds."
+  [ms func]
   (js/setInterval func ms))
 
-(defn wait [ms func]
+(defn wait
+  "Wait `ms` milliseconds before executing `func`."
+  [ms func]
   (js/setTimeout func ms))
 
-(defn now []
+(defn now
+  "Return the current time in milliseconds starting from the Unix epoch."
+  []
   (.getTime (js/Date.)))
```

Comment:
Thanks for the tip, I've updated the things to [refactor wiki page](https://github.com/LightTable/LightTable/wiki/Possible-Refactoring-Targets) with this info.